qiskit/providers/ibmq/runtime/utils.py

```diff
@@ -31,6 +34,14 @@ def default(self, obj: Any) -> Any:  # pylint: disable=arguments-differ
             return {'__type__': 'array', '__value__': obj.tolist()}
         if isinstance(obj, complex):
             return {'__type__': 'complex', '__value__': [obj.real, obj.imag]}
+        if isinstance(obj, QuantumCircuit):
+            # Serialize, compress, encode.
+            qpy_buffer = io.BytesIO()
+            qpy_serialization.dump(qpy_buffer, obj)
```

Comment:
This just swapped in https://github.com/Qiskit/qiskit-terra/commit/247f44ef87b08302514e512e4ed36601e95f33cd to be consistent with pickles order (it was tripping up the users testing qpy)

Reply:
Fixed in a4a84df